Server/src/server/gameCenter/models/game/Game.java

```diff
@@ -177,10 +177,12 @@ public void setNewNextCard() {
 
     public void replaceCard(String cardID) throws LogicException {
         if (getCurrentTurnPlayer().getCanReplaceCard()) {
+            System.out.println("server allows the replace ");
```

Comment:
Generally speaking, when reading the console we are more interesting in when the code fails rather than when it is successful.    Make the printout tells about the time replace failed.

Reply:
This is a debugging statement I meant to remove